src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -499,15 +499,20 @@ public override void OnPartitionsAssigned(IImmutableSet<TopicPartition> partitio
                 // _actor._consumer.Pause(partitionsToPause);
                 
                 _actor._commitRefreshing.AssignedPositions(partitions, _actor._consumer, _actor._settings.PositionTimeout);
-                // TODO: Add warning if IPartinionEventHandler will be public and call takes more then _warningDuration
-                _partitionEventHandler.OnAssign(partitions);
+
+                var watch = Stopwatch.StartNew();
+                _partitionEventHandler.OnAssign(partitions, _restrictedConsumer);
+                CheckDuration(watch, "onAssign");
+                
                 _actor._rebalanceInProgress = false;
             }
 
             public override void OnPartitionsRevoked(IImmutableSet<TopicPartitionOffset> partitions)
             {
-                // TODO: Add warning if IPartinionEventHandler will be public and call takes more then _warningDuration
-                _partitionEventHandler.OnRevoke(partitions);
+                var watch = Stopwatch.StartNew();
+                _partitionEventHandler.OnRevoke(partitions, _restrictedConsumer);
+                CheckDuration(watch, "onRevoke");
+                
```

Comment:
@IgorFedchenko do we need to `.Dispoe()` `Stopwatch` here?

Reply:
It does not implement `IDisposable` interface, but I do `watch.Stop()` in `CheckDuration` method implementation. 
But while technically this is correct, this looks unobvious - I thought about that when was writing this, but decided to save coding lines :) Added explicit `Stop()` call before passing `watch` to `CheckDuration` method, now should look better.

Reply:
Ah got it, thanks for explaining